### ISSUE TITLE
possible bug.... maybe ..???

### ISSUE THREAD

**Issue by mahfuzullah** [Opticos/GWSL-Source#90]
some apps are not working and these apps are not listed in gwsl apps section

for example,
`(base) galib@DESKTOP-C4U0A05:/mnt/c/Users/mahfu$ snap-store` 

returns 
`Unable to init server: Could not connect: Connection refused
16:28:15:0397 Gtk cannot open display:`

**Comment by Pololot64** [comment]
As for some apps not showing in the apps section it is because they do not all have .desktop files. I do not really plan on "fixing" this because there are too many other standards.

As for the snap store bugging out, do any other apps work?

**Comment by mahfuzullah** [comment]
> As for some apps not showing in the apps section it is because they do not all have .desktop files. I do not really plan on "fixing" this because there are too many other standards.
> 
> As for the snap store bugging out, do any other apps work?

and for gtk cant open display ???

**Comment by Pololot64** [comment]
Let me rephrase, for the gtk issue, does it only occur for the snap store or does it happen with different apps. Also, what is the output when you run echo $DISPLAY

**Comment by mahfuzullah** [comment]
> Let me rephrase, for the gtk issue, does it only occur for the snap store or does it happen with different apps. Also, what is the output when you run echo $DISPLAY

there is nothing just an empty line in the terminal

**Comment by Pololot64** [comment]
did you click the export display button in the GWSL dashboard?

**Comment by mahfuzullah** [comment]
> did you click the export display button in the GWSL dashboard? What does your .profile file look like?

exporter the display and pulse server everything working okay now, thanks

**Comment by Pololot64** [comment]
Glad it works. Have a good night/day.
